tests/unittests/BackendTest.cpp

```diff
@@ -165,6 +165,77 @@ TEST(RuntimeBundle, BundleSymbolInfo) {
   EXPECT_EQ(table.find("tensorview_reshape")->second.output, false);
 }
 
+// Test if the placeholders are allocated contiguously as
+// Input|InputOutput|Output.
+TEST(RuntimeBundle, ContiguousPlaceholder) {
+  ExecutionEngine EE;
+  auto &MD = EE.getModule();
+  auto *F = MD.createFunction("main");
+
+  unsigned depth = 16;
```

Comment:
There is no need in a complex use-case involving convolutions and FCs for this PR. It just adds complexity. Use something simpler. Any operations (e.g. add, sub, etc) would do.

Reply:
> There is no need in a complex use-case involving convolutions and FCs for this PR. It just adds complexity. Use something simpler. Any operations (e.g. add, sub, etc) would do.

OK. I just simplified the network.